src/PairedUrl.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Class PairedUrls.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP;
+
+use AmpProject\AmpWP\Infrastructure\Service;
+
+/**
+ * Service for manipulating a paired URL.
+ *
+ * @package AmpProject\AmpWP
+ * @since 2.1
+ * @internal
```

Comment:
This class is needed as a constructor argument when creating a custom `PairedUrlStructure` implementation, so it shouldn't be marked as `@internal`.
```suggestion
```